src/include/sof/ipc.h

```diff
@@ -202,14 +202,25 @@ int ipc_dma_trace_send_position(void);
 /* get posn offset by pipeline. */
 int ipc_get_posn_offset(struct ipc *ipc, struct pipeline *pipe);
 
+enum sof_pm_state {
+	SOF_PM_D0 = 0,	/* Transition to D0, also initial state */
```

Comment:
probably best to call the prefix all enum values with enum name. i.e. SOF_PM_STATE_D0 etc

Reply:
change applied